TradeItIosTicketSDK2/TradeItLinkBrokerUIFlow.swift

```diff
@@ -46,10 +49,12 @@ class TradeItLinkBrokerUIFlow: NSObject, TradeItWelcomeViewControllerDelegate, L
         onNavigationController navController: UINavigationController,
         asRootViewController: Bool,
         showWelcomeScreen: Bool,
+        hideOpenAccountButton: Bool,
```

Comment:
it was just to make the if more readable and I assumed by default it will be shown:
```
if hideOpenAccountButton {
     self.openNewAccountButton.removeFromSuperview()
 }
```